test/tools/api.js

```diff
@@ -80,38 +80,42 @@ const asyncApi = [
   { className: 'Collection', method: 'updateOne', returnType: 'Promise<UpdateResult>' },
+  { className: 'Collection', method: 'initializeOrderedBulkOp', returnType: 'OrderedBulkOperation', special: 'not a callback accepting API' },
+  { className: 'Collection', method: 'initializeUnorderedBulkOp', returnType: 'UnorderedBulkOperation', special: 'not a callback accepting API' },
 
-  { className: 'Db', method: 'addUser', returnType: 'Promise<Document>' },
-  { className: 'Db', method: 'collections', returnType: 'Promise<Collection[]>' },
+  { className: 'Db', method: 'addUser', returnType: 'Promise<Document>', special: 'see Admin.addUser' },
+  { className: 'Db', method: 'collections', returnType: 'Promise<Collection[]>', special: 'returns transformed collections' },
   { className: 'Db', method: 'command', returnType: 'Promise<Document>' },
-  { className: 'Db', method: 'createCollection', returnType: 'Promise<Collection<TSchema>>' },
+  { className: 'Db', method: 'createCollection', returnType: 'Promise<Collection<TSchema>>', special: 'returns transformed collection' },
   { className: 'Db', method: 'createIndex', returnType: 'Promise<string>' },
   { className: 'Db', method: 'dropCollection', returnType: 'Promise<boolean>' },
   { className: 'Db', method: 'dropDatabase', returnType: 'Promise<boolean>' },
   { className: 'Db', method: 'indexInformation', returnType: 'Promise<Document>' },
   { className: 'Db', method: 'profilingLevel', returnType: 'Promise<string>' },
   { className: 'Db', method: 'removeUser', returnType: 'Promise<boolean>' },
-  { className: 'Db', method: 'renameCollection', returnType: 'Promise<Collection<TSchema>>' },
+  { className: 'Db', method: 'renameCollection', returnType: 'Promise<Collection<TSchema>>', special: 'returns transformed collection' },
   { className: 'Db', method: 'setProfilingLevel', returnType: 'Promise<ProfilingLevel>' },
   { className: 'Db', method: 'stats', returnType: 'Promise<Document>' },
 
   { className: 'FindCursor', method: 'count', returnType: 'Promise<number>' },
   { className: 'FindCursor', method: 'explain', returnType: 'Promise<Document>' },
 
-  { className: 'GridFSBucket', method: 'delete', returnType: 'Promise<void>' },
+  { className: 'GridFSBucket', method: 'delete', returnType: 'Promise<void>', possibleCallbackPositions: [1] },
   { className: 'GridFSBucket', method: 'drop', returnType: 'Promise<void>' },
-  { className: 'GridFSBucket', method: 'rename', returnType: 'Promise<void>' },
+  { className: 'GridFSBucket', method: 'rename', returnType: 'Promise<void>', possibleCallbackPositions: [1] },
+  { className: 'GridFSBucket', method: 'openUploadStream', returnType: 'GridFSBucketWriteStream', special: 'returns transformed GridFSBucketWriteStream' },
+  { className: 'GridFSBucket', method: 'openUploadStreamWithId', returnType: 'GridFSBucketWriteStream', special: 'returns transformed GridFSBucketWriteStream' },
```

Comment:
what does this description in "special" mean?

Reply:
removed